src/gov-action-contracts/AIPs/MiscAIP/UpdateL1CoreTimelockAction.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import "../../address-registries/L1AddressRegistry.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+contract UpdateL1CoreTimelockAction {
+    ProxyAdmin public immutable govProxyAdmin;
```

Comment:
Maybe we should include proxy admins on the address registries in future?

Reply:
yeah that'd be good (outside of scope here tho)